src/machinery/eventListener.ts

```diff
@@ -14,14 +14,16 @@ let selection = undefined;
 
 const elementSelector = (selectedElement) => {
   selection = selectedElement;
-  console.log('Selected: ' + selectedElement);
 };
-let navigation = new Navigation([], elementSelector);
+let navigation = null;
```

Comment:
It's good practice to have explicit types for this: `let navigation: Navigation | null = null`

Reply:
Yeahh, true that. But if it is empty there are a couple of problems. I mean I can check it. Okay, I check inside the class, if it is empty.

Reply:
@aspic Fixed it.